pkg/helm/actions/list_releases_test.go

```diff
@@ -35,4 +35,9 @@ func TestListReleases(t *testing.T) {
 		t.Error(err.Error())
 	}
 	assert.Len(t, rels, 1, "Release list should return 1 release")
+	if len(rels) > 0 {
```

Comment:
no need for if, because at this point it must be 1, otherwise it would fail on the previous line.

Reply:
removed